addons/account/static/src/components/section_and_note_fields_backend/section_and_note_backend.scss

```diff
@@ -2,40 +2,65 @@
 // The goal of this file is to contain CSS hacks related to allowing
 // section and note on sale order and invoice.
 
-table.o_section_and_note_list_view tr.o_data_row {
-    &.o_is_line_note {
-        font-style: italic;
-    }
-    &.o_is_line_section {
-        font-weight: bold;
-        background-color: #DDDDDD;
-        --table-bg: var(--300);
-    }
-    &.o_is_line_subsection {
-        --table-bg: var(--200);
-    }
-    &.o_is_line_section, &.o_is_line_subsection {
-        border-top: 1px solid #BBB;
-        border-bottom: 1px solid #BBB;
-        .o_list_section_options {
-            width: 1px;  // to prevent the column to expand
-            button {
-                padding: 0px;
-                background: none;
-                border-style: none;
-                display: table-cell;
-                cursor: pointer;
+table.o_section_and_note_list_view {
+    --o-section-border-color: #{$gray-500};
+    --o-section-hover-effect-transition: 0.1s ease-in-out;
```

Comment:
`-hover`: Misleading. This variable is not used only for `:hover`. These rules are applied on the base selectors, thus are triggered for any state.
`-effect`: not necessary.

AFAIK, this value is used twice and never change contextually. You can create a scoped scss variable instead. Even better, you may use the value from `$transition-base` or `$transition-fade`. Eg:

```scss
$_SectionAndNote-transition-base: $transition-base;
```